.gitignore

```diff
@@ -1,9 +1,9 @@
-*.gem
+.ruby-gemset
+.ruby-version
 *.rbc
 .bundle
 .config
 .yardoc
-Gemfile.lock
```

Comment:
Is vagrant using the lockfile when resolving the dependencies? I guess having it in the repo only makes sense if it does.